docs/docs-content/audit-logs/monitor-with-datadog.md

```diff
@@ -0,0 +1,167 @@
+    | **datadog.networkMonitoring.enabled**        | Set it to `true` to enable network performance monitoring.                                                                                                                                                        |
+    | **datadog.serviceMonitoring.enabled**        | Set it to `true` to enable universal service monitoring.                                                                                                                                                          |
+    | **datadog.securityAgent.compliance.enabled** | Set it to `true` to enable the security agent.                                                                                                                                                                    |
+
+    Paste the following snippet under the `clusterAgent.volumes` section to enable the volume and volume mount for
+    collecting audit logs. Replace `/var/log/apiserver` with the `audit-log` path configured in the Kubernetes layer of
+    your host cluster. You can find this path in the Kubernetes layer of your host cluster profile in Palette.
+
+    ```yaml {3,13}
+    volumes:
+      - hostPath:
+          path: /var/log/apiserver
+        name: auditdir
+      - name: dd-agent-config
+        configMap:
+          name: dd-agent-config
+          items:
+            - key: kubernetes-audit-log
+              path: conf.yaml
+    volumeMounts:
+      - name: auditdir
+        mountPath: /var/log/apiserver
+      - name: dd-agent-config
+        mountPath: /conf.d/kubernetes_audit.d
+    ```
+
+    Additionally, if your host cluster uses Kubernetes version 1.25 or later, add the following lines under the
+    `pack.namespace` section to accommodate the elevated privileges required by the Datadog pods. Replace
+    `<k8s_version>` with your cluster's Kubernetes version and only include the major and minor version following the
+    lowercase letter **v**. For example, `v1.30`. Refer to the
+    [Deployment Violates Pod Security](../troubleshooting/cluster-deployment.md#deployment-violates-pod-security)
+    troubleshooting guide for more information.
+
+    ```yaml {3-4}
+    pack:
+      namespace: "datadog"
+      namespaceLabels:
+        "datadog": "pod-security.kubernetes.io/enforce=privileged,pod-security.kubernetes.io/enforce-version=v<k8s_version>"
+    ```
+
+    Once you finish making the necessary changes, click **Confirm & Create**, then select **Next**.
+
+12. Review the cluster profile and click **Finish Configuration**.
+
+13. In Palette, attach the Datadog add-on cluster profile to your host cluster. Refer to the
+    [Attach an Add-on Profile](../clusters/imported-clusters/attach-add-on-profile.md) guide for instructions.
+
+14. Create a ConfigMap resource for the host cluster to collect data. Open a terminal window and create a file named
```

Comment:
Switch the order. Start by telling the user to open a terminal, and then create the configMap.  Otherwise it feels a bit out of order.

Reply:
:white_check_mark: